### ISSUE TITLE
BeVSLaterality contains two inactive concepts

### ISSUE THREAD

**Issue by pedro-earlytracks** [hl7-be/core-clinical#46]
The following concepts in [BeVSLaterality ](https://www.ehealth.fgov.be/standards/fhir/core-clinical/ValueSet-be-vs-laterality.html)are inactive:

419161000 |Unilateral left (qualifier value)| 

419465000 |Unilateral right (qualifier value)|)

They should be replaced with: 

7771000 |Left (qualifier value)|
24028007 |Right (qualifier value)|

**Comment by annabel-uzl** [comment]
Indeed

**Comment by bdc-ehealth** [comment]
@mlambot 

This valueset should be managed by the NRC and not by the FHIR profiles. @jmp-ehealth, this is an element that should be taken into account in case of a new publication.

**Comment by davidodb** [comment]
@bdc-ehealth, @jmp-ehealth  I agree, the value set should be hosted and published by the NRC. And also maintained regarding inactivation of concepts.
Regarding the "management" of the content of the sets in general, it is up to the "domain experts" to define the content and apply/request eventual changes after publication.
Other general remarks: the purpose and use case for every value set needs to be 100% clear; try to use intentional value sets as much as possible in the future.
But for now, yes, we as NRC should publish and maintain the value set in question once everyone is 100% clear and agrees on the content and use case.

**Comment by KS-FPSHEALTH** [comment]
Hi all,

Looking purely at laterality, I agree with Marie Alexandra to define this intentionally using the query <182353008 | Side (qualifier value)>.

However, this means you will lose the other 'position-defining' values that were included in the initial value list. Is it necessary to retain these values and therefore define an additional reference set, or not—depending on the use case?

And a more general question: who is responsible for validating the values included in these reference sets? (short term and long term)
